diagnostic.md

```diff
@@ -66,5 +81,10 @@ arguments. Write the code to include the mixin inside all elements with a
 class of `content`.
 
 ```scss
-// your response here
+.content {
+    @mixin row() {
+      @include
+    }
```

Comment:
Not quite. When you use `@include`, you just put it into the curly braces for the selector you want to modify.
i.e.
```
.content {
  @include row;
}
```